frontend/packages/vkt/src/components/publicEnrollment/PublicEnrollmentStepper.tsx

```diff
@@ -55,17 +55,23 @@ export const PublicEnrollmentStepper = ({
   const isStepCompleted = (stepNumber: number) =>
     stepNumber < activeStep && !(isError && stepNumber === doneStepNumber);
 
-  const mobileStepValue = activeStep * (100 / doneStepNumber);
-  const mobilePhaseText = `${activeStep}/${doneStepNumber}`;
+  const stepValue =
+    activeStep >= PublicEnrollmentFormStep.Done
+      ? PublicEnrollmentFormStep.Done
+      : activeStep;
+
+  const mobileStepValue = stepValue * (100 / PublicEnrollmentFormStep.Done);
+  const mobilePhaseText = `${stepValue}/${PublicEnrollmentFormStep.Done}`;
```

Comment:
Näissä pitäis nähdäkseni vaihtaa `PublicEnrollmentFormStep.Done` tilalle `doneStepNumber` ja osoittajassa `stepValue`:n sijaan pitäisi arvon olla niin ikään `doneStepNumber` mikäli Preview-vaihe ollaan ohitettu.

Reply:
Jos tää ja/tai ylempi kommentti ei tarpeeksi selvä niin voin koittaa selventää tai sit toteuttaa näitä vaikka lokaalisti.